src/libraries/System.Private.CoreLib/src/System/IO/File.cs

```diff
@@ -1007,5 +1007,40 @@ public static Task AppendAllLinesAsync(string path, IEnumerable<string> contents
                 ? Task.FromCanceled(cancellationToken)
                 : InternalWriteAllLinesAsync(AsyncStreamWriter(path, encoding, append: true), contents, cancellationToken);
         }
+
+        /// <summary>
+        /// Creates a file symbolic link identified by <paramref name="path"/> that points to <paramref name="pathToTarget"/>.
+        /// </summary>
+        /// <param name="path">The path where the symbolic link should be created.</param>
+        /// <param name="pathToTarget">The path of the target to which the symbolic link points.</param>
+        /// <returns>A <see cref="FileInfo"/> instance that wraps the newly created file symbolic link.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="path"/> or <paramref name="pathToTarget"/> is <see langword="null"/>.</exception>
+        /// <exception cref="ArgumentException"><paramref name="path"/> or <paramref name="pathToTarget"/> is empty.
+        /// -or-
+        /// <paramref name="path"/> is not an absolute path.
```

Comment:
This is different from `File.Create`, which allows `path` to be relative.  I'm not necessarily opposed to requiring an absolute path, but it seems like it could be a gotcha.

Reply:
Implementation wise we do support relative paths, similarly to any other API in System.IO. This is just a mistake in the documentation. Thanks for pointing this out.

Reply:
My bad. I initially wrote the empty APIs with docs on top of them assuming some things, but didn't adjust them after writing the code. Thanks for catching it, @carlreinke. We'll make sure to double check the docs.